### ISSUE TITLE
Use /proxiesfor API to hook up proxy group support

### ISSUE THREAD

**Issue by cbeer** [sul-dlss/mylibrary#737]

**Comment by thatbudakguy** [comment]
we know that proxy groups are modeled differently in FOLIO, and this has UX implications/changes for the faculty views in myaccount. blocked on UX changes

**Comment by thatbudakguy** [comment]
this is ready for some investigation, at least, to discover how FOLIO differs from symphony

**Comment by cbeer** [comment]
I think FOLIO is generally better; loans have user id and a proxy user id fields, fees/fines appear to accrue to the patron user. Holds don't seem to have  anything to identify proxy vs personal, but there's always the request comment.

**Comment by marlo-longley** [comment]
I think via #827 we have discovered that there _is_ identifying info on holds for proxy vs personal (see #868) -- but we've learned that there is no concept of a group. I think we may need to open a new ticket or use this one to investigate how to change the UI related to what we see described in this test, given the absence of groups in FOLIO. I don't have access to seeing what this view looks like -- I am not a sponsor or a proxy in production/Symphony. But this reads to me like we will be losing a whole view.

https://github.com/sul-dlss/mylibrary/blob/99d9511a9d3f1c94121985a32073c0a39345c42f/spec/features/proxy_user_spec.rb#L4

**Comment by cbeer** [comment]
I'm not sure I understand "given the absence of groups in FOLIO"? The FOLIO model seems pretty similar to what we have in Symphony to me. The only wrinkle might be differentiating between the stuff the sponsor does individually vs on behalf of the group?

**Comment by marlo-longley** [comment]
This is highly related to #868 don't believe merging that PR will entirely close this issue, since it only deals with checkouts and requests, not fines or anything else.

**Comment by marlo-longley** [comment]
testing Github notifications with @saseestone

**Comment by marlo-longley** [comment]
Testing if she got notified without an explicit mention

**Comment by marlo-longley** [comment]
With #868 closed, we pulled out the fee/fine proxy work into #873 and can close this ticket.
